packages/odyssey-storybook/src/components/odyssey-labs/DatePicker/DatePicker.stories.tsx

```diff
@@ -46,7 +46,7 @@ export default storybookMeta;
 
 export const DatePickerStandard: StoryObj<DatePickerProps<unknown, unknown>> = {
   render: function C(props) {
-    const [value, setValue] = useState<unknown>(Date.now());
+    const [value, setValue] = useState<unknown>("09/05/1977");
```

Comment:
We should also change `unknown` to `string` since now we know it's a string.

Reply:
`value` is typed to `any` in MUI; maybe that's why it's `unknown` here?